package.json

```diff
@@ -8,11 +8,12 @@
   "scripts": {
     "test": "yarn install && yarn compile",
     "start": "yarn install && yarn compile && (test \"hi\" && (test -d server || yarn update-server)) || echo I can\\'t test whether the server exists! && electron .",
+    "start-win": "yarn install && yarn compile && (test \"hi\" && (test -d server || yarn update-server-win)) || echo I can\\'t test whether the server exists! && electron .",
     "start-dev": "yarn install && yarn compile && electron . --dev",
-
     "update-server": "git submodule update --remote && yarn compile-server",
+    "update-server-win": "git submodule update --remote && yarn compile-server-win",
     "compile-server": "cd socha && ./gradlew :server:deploy -q && cd .. && rimraf server && cp -r socha/server/build/runnable server && cp socha/player/build/libs/defaultplayer.jar .",
-
+    "compile-server-win": "cd socha && ./gradlew :server:deploy -q && cd .. && rimraf server && xcopy socha\\server\\build\\runnable server\\ /E && xcopy socha\\player\\build\\libs\\defaultplayer.jar .",
```

Comment:
ich möchte eigentlich nicht verschiedene scripts für verschiedene OS haben. Gibt es echt kein `cp -r` in Windows? Dann branche bitte, genau wie im `start`-command auch. Das ist so geschrieben, weil es "test" in Windows nicht gibt.

Reply:
Hm... das wusste ich nicht, hatte im Internet nur die Tricks mit :; goto etc. gefunden. Werde das anpassen